lib/shared/config.js

```diff
@@ -140,14 +140,8 @@ function normalize(config, options) {
                 .map(plugin => resolvePlugin(plugin, basedir))
                 .filter(Boolean);
 
-            switch (modelConfig.cache) {
```

Comment:
Looks like you changed the logic. What's the reason?

Reply:
You now always have default cache dir on the base config level, so we don't need to inherit this setting if model level cache not set.